### ISSUE TITLE
Add events for click on step label

### ISSUE THREAD

**Issue by niutech** [ahlechandre/mdl-stepper#37]
When using an editable stepper, I want to invoke form fields validation on leaving the current step. For now, I can handle it using events: `onstepnext` and `onstepback`. Please add a similar event on clicking the step label at the top, where I would be able to cancel moving to another step depending on my condition.

**Comment by fnicollet** [comment]
Agreed, another event, more generic like "onStepperChange", which could be triggered either when clicking the step label or the prev/next would be a nice addition to the existing events:
https://ahlechandre.github.io/mdl-stepper/component/

So far, you have to add several handler, including "onclick" events on labels to achieve that

Thanks,
